python/lsst/ts/standardscripts/maintel/take_triplet_comcam.py

```diff
@@ -0,0 +1,244 @@
+    def get_schema(cls) -> dict:
+        schema_yaml = """
+            $schema: http://json-schema.org/draft-07/schema#
+            $id: https://github.com/lsst-ts/ts_standardscripts/maintel/TakeTripletComCam.yaml
+            title: TakeTripletComCam v1
+            description: Configuration for TakeTripletComCam.
+            type: object
+            properties:
+              filter:
+                description: Filter name or ID; if omitted the filter is not changed.
+                anyOf:
+                  - type: string
+                  - type: integer
+                    minimum: 1
+                  - type: "null"
+                default: null
+              exposure_time:
+                description: The exposure time to use when taking images (sec).
+                type: number
+                default: 30.
+              dz:
+                description: De-focus to apply when acquiring the intra/extra focal images (microns).
+                type: number
+                default: 1500.
+              n_iterations:
+                description: Number of triplets.
+                type: integer
+                default: 1
+              program:
+                description: >-
+                    Optional name of the program this dataset belongs to.
+                type: string
+                default: TRIPLET
+              reason:
+                description: Optional reason for taking the data.
+                anyOf:
+                  - type: string
+                  - type: "null"
+                default: null
+            additionalProperties: false
+        """
+        return yaml.safe_load(schema_yaml)
+
+    async def configure(self, config: types.SimpleNamespace) -> None:
+        """Configure script.
+
+        Parameters
+        ----------
+        config : `types.SimpleNamespace`
+            Script configuration, as defined by `schema`.
+        """
+        # Configure tcs and camera
+        await self.configure_tcs()
+        await self.configure_camera()
+
+        # Set filter
+        self.filter = config.filter
+
+        # Set exposure time
+        self.exposure_time = config.exposure_time
+
+        # Set intra/extra focal offsets
+        self.dz = config.dz
+
+        # Set maximum number of iterations
+        self.n_iterations = config.n_iterations
+
+        # Set program and reason
+        self.reason = config.reason
+        self.program = config.program
+
+    def set_metadata(self, metadata: salobj.type_hints.BaseMsgType) -> None:
+        """Sets script metadata.
+
+        Parameters
+        ----------
+        metadata : `salobj.type_hints.BaseMsgType`
+            Script metadata topic. The information is set on the topic
+            directly.
+        """
+        # Estimated duration is maximum number of iterations multiplied by
+        # 3 (data triplet) multiplied by the time it takes to take an image
+        # plus estimation on processing the data (10s)
```

Comment:
Processing the data? or you mean reading out? Is there any processing going on in the script?

Reply:
No processing going on. Changed to reading out.